### PR TITLE
BGDIINF_SB-1427: Validate geometry fields

### PR DESCRIPTION
**Pull request by rebert**
A Validator has been defined, that validates geometries

### CODE REVIEW

**Review by hansmannj — APPROVED** [review]
:+1:
Nice, looks good to me

**Review by ltshb — COMMENTED** [review]
:+1:  very nice, but I would add some model unittest to make sure that it work as expected. Unittest should cover also boarder cases (see test_item_model.py as example).

File: app/stac_api/models.py [hunk 1]
```diff
@@ -67,6 +67,20 @@ def float_in(flt, floats, **kwargs):
     return np.any(np.isclose(flt, floats, **kwargs))
 
 
+def validate_valid_geometry(geometry):
```
Comment (boecklic) [inline app/stac_api/models.py:70]: Detail: `validate_geometry` is probably sufficient as function name, I think it's obvious that we don't want to validate for 'invalid' geometries ;-)

**Review by boecklic — APPROVED** [review]

File: app/tests/test_item_model.py [hunk 2]
```diff
@@ -127,3 +128,50 @@ def test_item_property_eo_gsd(self):
...
+
+    def test_item_create_model_empty_geometry(self):
+        # empty geometry should not be allowed
+        with self.assertRaises(ValidationError):
+            item = Item(
+                collection=self.collection,
+                properties_datetime=utc_aware(datetime.utcnow()),
+                item_name='item-empty', geometry=GEOSGeometry(
+                        'POLYGON EMPTY'
+                )
```
Comment (ltshb) [inline app/tests/test_item_model.py:165]: code style, don't forget to run `make format` or even better integrate yapf in your IDE

**Pull request merged**